### ISSUE TITLE
[Adaptive Cards-InputsWithValidation.json]: In high contrast Aquatic and Desert theme, focus indicator is not properly visible on the 'Accept' and 'Dismiss' icon controls present inside the 'Start' time flyout.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9231]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0

**Pre-requisites:**
1. Turn on 'Aquatic/Desert theme' (Navigation Path: System Settings> Accessibility > Contrast theme>Aquatic/Desert theme)

**Repro steps:**
1. Open the Adaptive Cards Visualizer application.
2. Navigate to the 'InputsWithValidation.json' control present on left navigation using tab key and activate it using enter key.
3. 'InputsWithValidation.json' adaptive card will appear on right sight of the pane.
4. Navigate to the 'Start time' control using tab key and activate using enter key.
5. Now navigate to the 'Accept' and 'Dismiss' icon controls using tab key.
6. Verify that, in high contrast Aquatic and Desert theme, focus indicator is properly visible on the 'Accept' and 'Dismiss' icon controls present inside the 'Start' time flyout or not.

**Actual Result:**
In high contrast Aquatic and Desert theme, focus indicator is not properly visible on the 'Accept' and 'Dismiss' icon controls present inside the 'Start' time flyout. Currently a static square boundary is visible for both the control due to which focus indicator is not visible on control as the focus indicator overlapped with the static boundary.

**Observation:**
1. Same issue replicates in 'Dusk/Night sky' themes (Navigation Path: System Settings> Accessibility > Contrast theme> 'Dusk/Night sky' theme)

**Expected Result:**
In high contrast Aquatic and Desert theme, a proper contrasting rectangular focus indicator should be clearly visible from all the four sides on the 'Accept' and 'Dismiss' icon controls present inside the 'Start' time flyout.

**User Impact:**
People with visual disabilities who rely on high contrast themes to see content properly will be impacted if in high contrast Aquatic and Desert theme, focus indicator is not properly visible on the 'Accept' and 'Dismiss' icon controls present inside the 'Start' time flyout due to which users will not be easily identify the currently focused control and might get disoriented on the page.

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/0399b786-52c0-44cc-97f0-9661b0531a2f

### Card JSON

```JSON
N/A
```

**Comment by vagpt** [comment]
Since the product is getting deprecated, hence closing this issue.
